src/main/java/org/folio/service/transactions/BatchTransactionService.java

```diff
@@ -0,0 +1,51 @@
+package org.folio.service.transactions;
+
+import io.vertx.core.Future;
+import io.vertx.ext.web.handler.HttpException;
+import org.folio.rest.core.model.RequestContext;
+import org.folio.rest.jaxrs.model.Batch;
+import org.folio.rest.jaxrs.model.Transaction;
+
+import java.util.List;
+
+public class BatchTransactionService {
```

Comment:
please don't forget to add logs to new service class

Reply:
Added.